src/components/content/credential/Credential.tsx

```diff
@@ -0,0 +1,228 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import React, { useEffect, useState } from 'react';
+import { ColumnsType } from 'antd/es/table';
+import { usernameKey } from '../../utils/constants';
+import AddCredential from './AddCredential';
+import UpdateCredential from './UpdateCredential';
+import { CredentialTip } from './CredentialTip';
+import CredentialDetails from './CredentialDetails';
+import { Button, FormInstance, Modal, Popconfirm, Space, Table } from 'antd';
+import { FullscreenOutlined, InfoCircleOutlined, MinusCircleOutlined, PlusCircleOutlined } from '@ant-design/icons';
+import {
+    CreateCredential,
+    CredentialsManagementService,
+    CredentialVariable,
+    CredentialVariables,
+} from '../../../xpanse-api/generated';
+
+function Credential(): JSX.Element {
+    const [isAddOpen, setIsAddOpen] = useState(false);
+    const [isUpdateOpen, setIsUpdateOpen] = useState(false);
+    const [isDetailsOpen, setIsDetailsOpen] = useState(false);
+    const formRef = React.createRef<FormInstance<CredentialVariables>>();
+    const [tipMessage, setTipMessage] = useState<string>('');
+    const [tipType, setTipType] = useState<'error' | 'success' | undefined>(undefined);
+    const [credentialDefinitionList, setCredentialDefinitionList] = useState<CredentialVariables[]>([]);
+    const [credentialDetails, setCredentialDetails] = useState<CredentialVariable[]>([]);
+    const [createCredential, setCreateCredential] = useState<CreateCredential>({
+        csp: '' as CredentialVariables.csp,
+        description: '',
+        name: '',
+        timeToLive: 0,
+        type: '' as CredentialVariables.type,
+        variables: [],
+        xpanseUser: '',
+    });
+
+    const columns: ColumnsType<CredentialVariables> = [
+        {
+            title: 'Csp',
+            dataIndex: 'csp',
+        },
+        {
+            title: 'User',
```

Comment:
Currently in the 'end user' menu, the credential with 'user' empty must not be displayed. 
The first screen shot in your PR shows that. We must filter out those default credentials.